lib/PuppeteerSharp/LaunchOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PuppeteerSharp
+{
+    public class LaunchOptions
+    {
+        public bool AppMode { get; set; }
+
+        public bool IgnoreHTTPSErrors { get; set; }
+
+        public bool IgnoreDefaultArgs { get; set; }
+
+        public bool Headless { get; set; } = true;
+
+        public string ExecutablePath { get; set; }
+
+        public int SlowMo { get; set; }
+
+        public string[] Args { get; set; } = Array.Empty<string>();
+
+        public bool HandleSIGINT { get; set; } = true;
+
+        public bool HandleSIGTERM { get; set; } = true;
+
+        public bool HandleSIGHUP { get; set; } = true;
+
+        public int Timeout { get; set; } = 30_000;
+
+        public bool Dumpio { get; set; }
+
+        public string UserDataDir { get; set; }
+
+        public IDictionary<string, string> Env { get; } = new Dictionary<string, string>();
+
+        public bool Devtools { get; set; }
+
+        public int KeepAliveInterval { get; set; } = 30;
```

Comment:
I didn't find this in the source code of https://github.com/GoogleChrome/puppeteer/blob/master/lib/Launcher.js#L150 but I didn't want to break the code

Reply:
`KeepAliveInterval` is being used by the WebSocket. https://github.com/kblok/puppeteer-sharp/blob/2dc3661299a8574ba5abb7ba020d55629df7079c/lib/PuppeteerSharp/Connection.cs#L196

30 seconds should be enough. The unit test is setting this value to 120 seconds because you might need to pause and debug the code.